config/mapbox-codestyle.xml

```diff
@@ -0,0 +1,298 @@
+    <option name="WHILE_BRACE_FORCE" value="1" />
+    <option name="METHOD_ANNOTATION_WRAP" value="1" />
+    <option name="CLASS_ANNOTATION_WRAP" value="1" />
+    <option name="FIELD_ANNOTATION_WRAP" value="1" />
+    <option name="PARAMETER_ANNOTATION_WRAP" value="1" />
+    <option name="VARIABLE_ANNOTATION_WRAP" value="1" />
+    <option name="ENUM_CONSTANTS_WRAP" value="1" />
+    <AndroidXmlCodeStyleSettings>
+        <option name="USE_CUSTOM_SETTINGS" value="true" />
+        <option name="LAYOUT_SETTINGS">
+            <value>
+                <option name="INSERT_BLANK_LINE_BEFORE_TAG" value="false" />
+                <option name="INSERT_LINE_BREAK_AFTER_LAST_ATTRIBUTE" value="true" />
+            </value>
+        </option>
+    </AndroidXmlCodeStyleSettings>
+    <GroovyCodeStyleSettings>
+        <option name="ALIGN_MULTILINE_LIST_OR_MAP" value="false" />
+        <option name="ALIGN_NAMED_ARGS_IN_MAP" value="false" />
+    </GroovyCodeStyleSettings>
+    <JavaCodeStyleSettings>
+        <option name="CLASS_NAMES_IN_JAVADOC" value="3" />
+        <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
+        <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
+        <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
+            <value />
+        </option>
+        <option name="IMPORT_LAYOUT_TABLE">
+            <value>
+                <package name="" withSubpackages="true" static="false" />
+                <emptyLine />
+                <package name="" withSubpackages="true" static="true" />
+            </value>
+        </option>
+        <option name="JD_ALIGN_PARAM_COMMENTS" value="false" />
+        <option name="JD_ALIGN_EXCEPTION_COMMENTS" value="false" />
+        <option name="JD_P_AT_EMPTY_LINES" value="false" />
+        <option name="JD_DO_NOT_WRAP_ONE_LINE_COMMENTS" value="true" />
+        <option name="JD_KEEP_EMPTY_PARAMETER" value="false" />
+        <option name="JD_KEEP_EMPTY_RETURN" value="false" />
+        <option name="JD_PRESERVE_LINE_FEEDS" value="true" />
+    </JavaCodeStyleSettings>
+    <JetCodeStyleSettings>
+        <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+            <value />
+        </option>
+        <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="999" />
+        <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="999" />
+        <option name="IMPORT_NESTED_CLASSES" value="true" />
+        <option name="CONTINUATION_INDENT_IN_PARAMETER_LISTS" value="false" />
+        <option name="CONTINUATION_INDENT_FOR_EXPRESSION_BODIES" value="false" />
+        <option name="WRAP_EXPRESSION_BODY_FUNCTIONS" value="1" />
+        <option name="IF_RPAREN_ON_NEW_LINE" value="true" />
+    </JetCodeStyleSettings>
+    <XML>
+        <option name="XML_ALIGN_ATTRIBUTES" value="false" />
+        <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
+    </XML>
+    <ADDITIONAL_INDENT_OPTIONS fileType="php">
```

Comment:
Remove PHP?

Reply:
Yeah I don't think we need any of these indent options for these file types since we don't have any `.php`, `.scala` related code.
Wdyt?

Reply:
Yeah let's remove 🚀